cumulusci/tasks/bulkdata/__init__.py

```diff
@@ -1,7 +1,15 @@
-from cumulusci.tasks.bulkdata.delete import DeleteData  # noqa: F401
+from cumulusci.tasks.bulkdata.delete import DeleteData
 from cumulusci.tasks.bulkdata.extract import ExtractData
-from cumulusci.tasks.bulkdata.generate import GenerateMapping  # noqa: F401
-from cumulusci.tasks.bulkdata.load import LoadData  # noqa: F401
+from cumulusci.tasks.bulkdata.generate import GenerateMapping
+from cumulusci.tasks.bulkdata.load import LoadData
+from cumulusci.tasks.bulkdata.generate_and_load_data import GenerateAndLoadData
 
 # For backwards-compatibility
 QueryData = ExtractData
+__all__ = (
+    "DeleteData",
+    "ExtractData",
+    "GenerateMapping",
+    "LoadData",
+    "GenerateAndLoadData",
+)
```

Comment:
It actually is more complicated than that. I spent like 20 minutes yesterday researching how to do that. Flake8 can only put comments on the same line as the stuff it refers to. Because the class and module names are so long, Black is turning this into a multi-line comment and preventing the Flake8 comments from working. The recommended fix in a VERY long and contentious thread I read is __all__.

Reply:
Okay, fair enough.

(I didn't know before that flake8 was smart enough to recognize the use as a string in `__all__`, so that's nice to learn.)